using_the_compiler/README.md

```diff
@@ -360,3 +360,12 @@ crystal tool format [--check] [<path>...]
 
 `path` can be a file or folder name and include all Crystal files in that folder tree. Omitting `path` is equal to
 specifying the current working directory.
+
+## Environment variables
+
+The following environment variables are used by the Crystal compiler if set in the environment. Otherwise the compiler will populate them with default values.. Their values can be inspected using `[crystal env](#crystal env)`.
+
+* `CRYSTAL_CACHE_DIR`: Defines path where Crystal caches partial compilation results for faster subsequent builds. This path is also used to temporarily store executables when Crystal programs are run with 'crystal run' rather than 'crystal build'.
```

Comment:
Maybe link the commands?
```suggestion
* `CRYSTAL_CACHE_DIR`: Defines path where Crystal caches partial compilation results for faster subsequent builds. This path is also used to temporarily store executables when Crystal programs are run with `[crystal env](#crystal-run)` rather than `[crystal build](#crystal-build)`.
```